src/style.css

```diff
@@ -0,0 +1,127 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+:root {
+  --white-color: #fff;
+  --dark-blue-color: #060b35;
+  --gray-color: #616070;
+  --light-blue-color: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.page {
+  margin: 0;
+}
+
+.card {
```

Comment:
Create separate files for card and stars blocks
![image](https://github.com/mate-academy/layout_product-cards/assets/96409563/3154a7ad-746f-4e71-b208-28c2b24ce172)